pkg/subctl/cmd/gather/resource.go

```diff
@@ -26,27 +27,32 @@ import (
 )
 
 func ResourcesToYAMLFile(info *Info, ofType schema.GroupVersionResource, namespace string) error {
-	list, err := info.DynClient.Resource(ofType).Namespace(namespace).List(metav1.ListOptions{})
+	lists, err := info.DynClient.Resource(ofType).Namespace(namespace).List(metav1.ListOptions{})
 	if err != nil {
 		return errors.WithMessagef(err, "error listing %q", ofType.Resource)
 	}
-
-	path := filepath.Join(info.DirName, info.ClusterName+"-"+ofType.Resource+".yaml")
-	file, err := os.Create(path)
-	if err != nil {
-		return errors.WithMessagef(err, "error opening file %s", path)
+	if len(lists.Items) == 0 {
+		return errors.New(fmt.Sprintf("No config maps found in namespace %s", namespace))
```

Comment:
```suggestion
		return errors.New(fmt.Sprintf("No %s found in namespace %s", ofType.Resource, namespace))
```

But may be this should simply be a warning instead?